trino/client.py

```diff
@@ -392,6 +408,13 @@ def http_headers(self) -> Dict[str, str]:
             for name, value in self._client_session.properties.items()
         )
 
+        if len(self._client_session.prepared_statements) != 0:
+            # ``name`` must not contain ``=``
```

Comment:
again a copy - proper assertion will be added + quoting/unquoting of name as well since that is what Trino server does.